lib/messages.js

```diff
@@ -103,6 +103,49 @@ exports.connectRelay = {
   }
 }
 
+exports.mutableReq = {
```

Comment:
nit: Request instead of Req as it's an exposes property

Reply:
I called it mutablePut at first but it looks weird because it's used in `onget` and `mutableGet` - that's why I went to Req